client/lib/signup/hint-data.js

```diff
@@ -1,673 +1,677 @@
-	'Teaching',
-	'Tech',
-	'Tech blog',
-	'Technical recruiting',
-	'Technical support',
-	'Technology',
-	'Telecommunications',
-	'Therapy',
-	'Thoughts',
-	'Time',
-	'Tips',
-	'Tourism',
-	'Tourism accomodation',
-	'Toys',
-	'Trading',
-	'Translation services',
-	'Transport',
-	'Travel',
-	'Travel agent',
-	'Traveller blog',
-	'Troubleshooting',
-	'Turorials',
-	'UI',
-	'UI designer',
-	'University',
-	'User interface',
-	'User interface design',
-	'VR',
-	'VR glasses',
-	'Vacation rental',
-	'Vacations',
-	'Vaccinations',
-	'Vaccines',
-	'Veterans',
-	'Veterans affairs',
-	'Video',
-	'Video games',
-	'Videography',
-	'Virtual assistant',
-	'Virtual reality',
-	'Walking',
-	'War',
-	'Warehouse',
-	'Watches',
-	'Watchmaking',
-	'Web design',
-	'Web designer',
-	'Web developer',
-	'Wedding',
-	'Wedding planner',
-	'Weight loss',
-	'Weightlifting',
-	'Well-being',
-	'Wellness',
-	'Window blinds',
-	'Wine',
-	'Women',
-	"Women's health",
-	'Words',
-	'Work performance',
-	'Workshop',
-	'World issues',
-	'World news',
-	'Writer',
-	'Writing',
-	'Yoga',
-	'Yoga for kids',
-	'Youtube videos',
-];
+export const hints = {
+	'3D': i18n.translate( '3D' ),
+	Abuse: i18n.translate( 'Abuse' ),
+	'Academic research': i18n.translate( 'Academic research' ),
+	Accessories: i18n.translate( 'Accessories' ),
+	Accountant: i18n.translate( 'Accountant' ),
+	'Accounting services': i18n.translate( 'Accounting services' ),
+	Acoustics: i18n.translate( 'Acoustics' ),
+	Acting: i18n.translate( 'Acting' ),
+	Activisim: i18n.translate( 'Activisim' ),
+	Adventure: i18n.translate( 'Adventure' ),
+	Advertising: i18n.translate( 'Advertising' ),
+	'Advocacy and human rights': i18n.translate( 'Advocacy and human rights' ),
+	'Advocacy group': i18n.translate( 'Advocacy group' ),
+	Advocate: i18n.translate( 'Advocate' ),
+	Affiliate: i18n.translate( 'Affiliate' ),
+	Agriculture: i18n.translate( 'Agriculture' ),
+	'Air conditioning': i18n.translate( 'Air conditioning' ),
+	Allah: i18n.translate( 'Allah' ),
+	Allergies: i18n.translate( 'Allergies' ),
+	Animals: i18n.translate( 'Animals' ),
+	Animation: i18n.translate( 'Animation' ),
+	Anime: i18n.translate( 'Anime' ),
+	Antiques: i18n.translate( 'Antiques' ),
+	Anxiety: i18n.translate( 'Anxiety' ),
+	'Anxiety and depression': i18n.translate( 'Anxiety and depression' ),
+	App: i18n.translate( 'App' ),
+	Apparel: i18n.translate( 'Apparel' ),
+	Architecture: i18n.translate( 'Architecture' ),
+	Archive: i18n.translate( 'Archive' ),
+	'Armed forces': i18n.translate( 'Armed forces' ),
+	Army: i18n.translate( 'Army' ),
+	Art: i18n.translate( 'Art' ),
+	'Art lessons': i18n.translate( 'Art lessons' ),
+	'Art school': i18n.translate( 'Art school' ),
+	Artist: i18n.translate( 'Artist' ),
+	Artists: i18n.translate( 'Artists' ),
+	Arts: i18n.translate( 'Arts' ),
+	Assistant: i18n.translate( 'Assistant' ),
+	Association: i18n.translate( 'Association' ),
+	Astronomy: i18n.translate( 'Astronomy' ),
+	Aurthor: i18n.translate( 'Aurthor' ),
```

Comment:
@fditrapani Can we remove this line or is it not a typo?

Reply:
It's a typo. Thanks for catching it. It should be `Author`